install_scripts/Dockerfile_Ubuntu

```diff
@@ -0,0 +1,54 @@
+# for OpenMined/Grid 
```

Comment:
It looks like there are two duplicate files, one named `Dockerfile_Ubuntu` and one named `Dockerfile_ubuntu`. Could you remove one as their duplicates?

Reply:
This might be the time I lost a PR and the file and I have to redo it. Anyway, I removed the duplicated/outdated one. The current one ("Dockerfile_Ubuntu") is the current one and the only one in the repo.

I have to said the one using Docker Compose (#136 ) seems more elegant, and I am open to suggestion to change.